recipes/yt-dlg/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "yt-dlg" %}
+{% set version = "1.8.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/yt_dlg-{{ version }}.tar.gz
+  sha256: 29340a44b90672e2e23fc96436117de2835c54ae7dc1845308b9cbf464e153f5
+
+build:
+  noarch: python 
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - yt-dlg = youtube_dl_gui.app:main
+
+requirements:
+  host:
+    - pip
+    - polib
+    - pyinstaller
+    - python >=3.7
+  run:
+    - polib >=1.1.0
+    - pyinstaller <=4.6,>=3.6
+    - pypubsub >=4.0.3
+    - python >=3.7
+    - wxpython <=4.1.2a1,>=4.0.7.post2
+
+test:
+  imports:
+    - youtube_dl_gui
+  commands:
+    - pip check
```

```suggestion
  commands:
    - pip check
    - yt-dlg --version
```
Test your entry points. On latest you can query the version and return.